views/components/collapsibleBlock.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * All About Olaf
+ * Individual row for inside a collapsible view
+ */
+
+import React from 'react'
+import {
+  View,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native'
+
+import Collapsible from 'react-native-collapsible'
+
+
+export default class CollapsibleBlock extends React.Component {
+
+  static propTypes = {
+    backgroundColor: React.PropTypes.string.isRequired,
```

Comment:
Here, a more general approach might be nice – what do you think about accepting a `style` property, instead of just a single `backgroundColor` prop? If you don't want to, that's fine – it's unlikely that we'll need to pass more styles down, but it might be nice ð